deepchem/molnet/load_function/bace_datasets.py

```diff
@@ -80,6 +108,21 @@ def load_bace_classification(featurizer=None, split='random'):
     )
 
   bace_tasks = ["Class"]
+  if save:

```

Comment:
These save functions are pretty similar for different datasets. Might be good to factor them out into a `dc.molnets.utils` file with a `save_dataset()` function that can be reused.